src/complete.jl

```diff
@@ -182,7 +188,7 @@ function complete_polynomial_impl!{T,N,D}(out::Type{Vector{T}}, z::Type{Vector{T
                             notD_d = notD_{d+1}*z[i_d]
                         end
 
-                        out[ix] = coeff_d * z[D]^(coeff_d-1) * notD_d
+                        out[ix] = coeff_d == 0 ? zero(T) : coeff_d * z[D]^(coeff_d-1) * notD_d
```

Comment:
Sorry I didn't totally follow. You said if `coeff_d ==0` then `coeff_d < 0` - How does that work?